src/toolkit/matl_sell_policy.cc

```diff
@@ -173,30 +197,37 @@ std::set<BidPortfolio<Material>::Ptr> MatlSellPolicy::GetMatlBids(
       req = *rit;
       qty = std::min(req->target()->quantity(), limit);
       bid_qty = excl ? quantize_ : package_->GetFillMass(qty);
-      if (bid_qty == 0) {
-        nbids = 0;
-      } else {
-        nbids = static_cast<int>(std::floor(qty / bid_qty));
+      if (bid_qty != 0) {
+        n_full_bids = static_cast<int>(std::floor(qty / bid_qty));
+        bids.assign(n_full_bids, bid_qty);
+
+        remaining_qty = fmod(qty, bid_qty);
```

Comment:
use `std::remainder` here to be consistent with above?

Reply:
actually I realized that they should all be `fmod` because the outcome should always be >0 which `remainder` does not guarantee